@utahdts/utah-design-system-header/src/js/renderables/footer/renderFooter.js

```diff
@@ -102,6 +102,20 @@ export default function renderFooter() {
         footerHr?.remove();
       }
 
+      // add custom privacy policy link
+      if (settings?.footer?.linkPrivacyPolicy) {
+        const privacyLink = document.getElementById(domConstants.FOOTER_LINK_PRIVACY_ID);
+        checkForError(!privacyLink, 'renderFooter: cannot find privacy policy link');
+        privacyLink?.setAttribute('href', settings.footer.linkPrivacyPolicy);
```

Comment:
is it possible the href was already set? some of the other onClick and other handlers check to make sure something wasn't already set before setting and throw an error if so... probably not pertinent here.